src/Integration/Binding/BindingProcessImpl.cs

```diff
@@ -143,6 +150,40 @@ public async Task<bool> DownloadQualityProfileAsync(IProgress<FixedStepsProgress
             return true;
         }
 
+        /// <summary>
```

Comment:
Please call out in a review comment when you've moved a chunk of code from one class to another and whether you've made any changes to it - it's hard to tell in the GitHub web UI.